rules/order.js

```diff
@@ -85,15 +174,23 @@ module.exports = {
         'line-height',
         'color',
         'text-align',
-        'white-space',
-        'vertical-align',
+        'text-indent',
+        'text-transform',
         'text-decoration',
+        'text-rendering',
+        'text-shadow',
+        'text-overflow',
+        'letter-spacing',
+        'word-spacing',
+        'word-wrap',
+        'word-break',
+        'white-space',
```

Comment:
Might make sense to move `overflow` down here, actually, you almost always want to set it when setting `white-space` an/or `text-overflow`.